packages/lisk-p2p/src/p2p_types.ts

```diff
@@ -64,11 +57,12 @@ export interface P2PNodeInfo {
 	readonly nethash: string;
 	readonly wsPort: number;
 	readonly height: number;
-	readonly options?: P2PInfoOptions;
+	// tslint:disable-next-line: no-mixed-interface
+	readonly [key: string]: unknown;
```

Comment:
if we are communicating through header/url is it possible to be `object`?

Reply:
Yes, it is possible that it could be an object, I mean we could still pass `options?` field, but now it can accept a wide range of types of values